module-release-meta.yaml

```
@@ -0,0 +1,10 @@
+apiVersion: operator.kyma-project.io/v1beta2
+kind: ModuleReleaseMeta
+metadata:
+  name: template-operator
+  namespace: kcp-system
+spec:
+  channels:
+    - channel: regular
+      version: 1.0.0
```

```suggestion
      version: 1.0.1
```

Lets use the latest version

---

But then we have to keep it aligned with the module-config.yaml as well

---

But then we have to keep it aligned with the module-config.yaml as well